packages/workspace/src/utils/workspace.ts

```diff
@@ -4,6 +4,27 @@ import {
   ProjectDefinition,
   TargetDefinition,
 } from '@angular-devkit/core/src/workspace';
+import { logger } from '@nrwl/tao/src/shared/logger';
```

Comment:
is this required?

Reply:
Oh shit webstorm auto imported the wrong logger.